apps/server/src/modules/board/controller/element.controller.ts

```diff
@@ -141,4 +143,18 @@ export class ElementController {
 
 		return response;
 	}
+
+	@ApiOperation({ summary: 'Check permission for a drawing element.' })
+	@ApiResponse({ status: 201 })
+	@ApiResponse({ status: 400, type: ApiValidationError })
+	@ApiResponse({ status: 403, type: ForbiddenException })
+	@ApiResponse({ status: 404, type: NotFoundException })
+	@HttpCode(201)
```

Comment:
201 seems wrong here. You don't create anything. https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

I'd say 200 is fine and then you don't need the @HttpCode decorator.